### ISSUE TITLE
[HOLD for payment 2024-06-20] [HOLD for payment 2024-06-18] [$250] Settings - When navigated to wallet page, blue bubble selection option is shown.

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#41197]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.67-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to https://staging.new.expensify.com/home
2. Tap settings
3. Long press and select the email id displayed under avatar
4. Tap Wallet

## Expected Result:
When navigated to wallet page, blue bubble selection option should not be shown.

## Actual Result:
When navigated to wallet page, blue bubble selection option is shown.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/4d8f3148-790f-4c96-a8b1-5021bda51f4d

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01626cae18fdbd316c</li>
        <li>Upwork Job ID: 1785760528249442304</li>
        <li>Last Price Increase: 2024-05-29</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @JmillsExpensify</details>

**Comment by izarutskaya** [comment]
We think this issue might be related to the #vip-vsb.

**Comment by JmillsExpensify** [comment]
Adding this to #vip-split and opening up to the community.

**Comment by sobitneupane** [comment]
Waiting Proposal

**Comment by JmillsExpensify** [comment]
Still waiting for proposals

**Comment by sobitneupane** [comment]
Thanks for the [proposal](https://github.com/Expensify/App/issues/41197#issuecomment-2117426022) @dominictb 

It looks like you're new to Expensify. Welcome!

> New contributors are limited to working on one job at a time, however experienced contributors may work on numerous jobs simultaneously.

As per the contributing guidelines, new contributor can only work on one issue at a time. 

Please let me know if you are assigned to a closed issue or an open issue with the `Awaiting payment` label.

**Comment by sobitneupane** [comment]
> Please let me know if you are assigned to a closed issue or an open issue with the `Awaiting payment` label.

@dominictb Any update?

**Comment by sobitneupane** [comment]
@dominictb I don't think we can technically allow that. I am quite surprised to see that you are already assigned to multiple issues. 

> So I think that one qualifies as an "Awaiting payment" issue since there's nothing else to be done from me

Please let me know once you fulfill the requirement.

**Comment by sobitneupane** [comment]
https://github.com/Expensify/App/issues/41197#issuecomment-2119256184

**Comment by JmillsExpensify** [comment]
Waiting for proposal update.

**Comment by sobitneupane** [comment]
Thanks @dominictb 

[Proposal](https://github.com/Expensify/App/issues/41197#issuecomment-2117426022) from @dominictb looks good to me.

🎀 👀 🎀 C+ reviewed

**Comment by amyevans** [comment]
Proposal sounds good to me! Assigning

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by AndrewGable** [comment]
As mentioned above, this caused a deploy blocker linked above. We are reverting the PR that fixed this issue, please test this regression case in a future PR when trying to fix the bug again.

**Comment by amyevans** [comment]
Thanks Andrew! @dominictb let us know how you get on with an updated proposal

**Comment by sobitneupane** [comment]
Thanks for the update @dominictb but I don't think we should go with the workaround.

It is a very rare bug. If there is no straight forward solution, I believe we can close this issue.

cc: @amyevans @JmillsExpensify

**Comment by amyevans** [comment]
I agree with @sobitneupane that the risks outweigh the rewards given the benefit of hindsight. I'd be in favor of closing without a follow up PR. I believe we'd still honor the $125 payouts since work was hired and completed ($250 with 50% regression penalty), but I'll defer to @JmillsExpensify for final say here!

**Comment by amyevans** [comment]
I'm unassigning since I'm headed out on extended leave after today. If you need input from another internal engineer feel free to do the 🎀 trick! 👋

**Comment by JmillsExpensify** [comment]
I agree with Amy. I've updated the payment summary above to reflect a payout of $125.

**Comment by JmillsExpensify** [comment]
@dominictb mind sharing your Upwork profile so that I can invite you to the Upwork job? The previous one expired.

**Comment by JmillsExpensify** [comment]
Offer sent

**Comment by JmillsExpensify** [comment]
All paid out! @sobitneupane please complete the checklist and request payments in NewDot.

**Comment by JmillsExpensify** [comment]
Whoops @dominictb I'm still waiting for you to accept the offer in Upwork.

**Comment by JmillsExpensify** [comment]
All paid out!

**Comment by sobitneupane** [comment]
**Regression Test Proposal**

1. Go to newDot on chrome browser on android (mWeb:Chrome)
2. Tap settings
3. Long press and select the email id displayed under avatar
4. Tap Wallet
5. Verify that blue bubble selection is not shown.

Do we agree 👍 or

**Comment by trjExpensify** [comment]
Confirming payment summary as follows: 

- $125 to @sobitneupane for the C+ review
- $125 to @dominictb for the fix

**Comment by JmillsExpensify** [comment]
$125 approved for @sobitneupane
